src/routes/tools/slr-coastal-inundation/index.svelte

```diff
@@ -232,7 +238,7 @@
       on:datasetLoaded="{(e) => datasetStore.set(e.detail)}"
     >
       <div slot="description">
-        <p>To do...</p>
+        {@html aboutContent}
```

Comment:
This feels like a better solution than having all the text here.

Reply:
Thanks @mukhtyar I wonder if we might be able to leverage it for other pieces of content such as in the "learn more" content modals?